src/phoenix/server/api/routers/v1/experiment_runs.py

```diff
@@ -39,7 +39,7 @@ async def create_experiment_run(request: Request) -> Response:
               trace_id:
                 type: string
                 description: Optional trace ID for tracking
-              output:
+              experiment_run_output:
                 type: string
                 description: The output of the experiment run
```

Comment:
`experiment_run_output` is a dictionary, isn't it?

Reply:
Would it make more sense to just return `output` here, with the output being `experiment_run_output.task_output`?

Reply:
Yes, that is an equivalent alternative, but consider the differences by looking at the diagram below. This PR is proposing the second transformation in the diagram. In the first transformation we are flattening the data structure into the same namespace. This somewhat limits the keywords that we can use in the future. It also adds an extra reorganizing step in the ingestion/retrieval process. One can argue that the server DB structure should just be flat to begin with. Yes, that's true, but net-net a flat structure is either equivalent or slightly worse: it does not appears to be strictly better than a nested structure.


<img width="611" alt="Screenshot 2024-07-02 at 6 07 27 AM" src="https://github.com/Arize-ai/phoenix/assets/80478925/1d6edc99-ef18-4fa7-96a6-09775b70d33e">